titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/KubeApiServerIntegrator.java

```diff
@@ -329,18 +329,10 @@ public void onDelete(V1Node node, boolean deletedFinalStateUnknown) {
 
     private void nodeUpdated(V1Node node) {
         try {
-            boolean removeStopped = node.getStatus().getConditions().stream()
```

Comment:
Do we depend on GC process here to do the cleanup?

Reply:
Yes.